test/unit/StakingPool/helpers.js

```diff
@@ -209,14 +209,18 @@ async function calculateStakeAndRewardsWithdrawAmounts(stakingPool, deposit, tra
   };
 }
 
-async function moveTimeToNextTranche(amountOfTranches) {
-  const { firstActiveTrancheId: currentTrancheId } = await getTranches();
-  const nextTrancheId = currentTrancheId + amountOfTranches;
+async function moveTimeToNextTranche(trancheCount) {
+  const nextTrancheId = (await getCurrentTrancheId()) + trancheCount;
   await setTime(nextTrancheId * TRANCHE_DURATION);
-
   return nextTrancheId;
 }
 
+async function moveTimeToNextBucket(bucketCount) {
```

Comment:
name could be improved here, since i understand it as moving to the next bucket (+1)

it's basically advancing time by bucketCount buckets

`advanceTimeByNBuckets`

`advanceTimeByBuckets`

Reply:
I just followed the naming already in place. The other function is used in a bunch of places so it's a won't fix for now.